bin/get-peers.sh

```diff
@@ -1,9 +1,24 @@
 #/bin/bash
-
 set -e
 
-# Get the configured cluster port
-cluster_port=$(lucky get-config cluster-port)
+# Make sure context parameter was given
+if [ -z ${1} ]; then
```

Comment:
Does `${1}` need to be in quotes: `"${1}"`. I think it is a syntax error if `${1}` it isn't provided in its current state.

We probably want the quotes on the if statements below as well.

Reply:
While it won't error, convention states it is probably a better way to write it. I'll change it.